client/components/App.jsx

```diff
@@ -1,19 +1,21 @@
 import React from 'react'
-import { Route, BrowserRouter as Router, Switch } from 'react-router-dom'
-import { Container } from 'semantic-ui-react'
+import {Route, BrowserRouter as Router, Switch} from 'react-router-dom'
+import {Container} from 'semantic-ui-react'
 
 import FishList from './FishList'
 import TopMenu from './TopMenu'
+import FishDetails from './FishDetails'
 
 class App extends React.Component {
   render () {
     return (
       <Router>
         <React.Fragment>
           <Route path='/' component={TopMenu} />
-          <Container style={{ marginTop: 75 }}>
+          <Container style={{marginTop: 75}}>
             <Switch>
               <Route exact path='/' component={FishList} />
+              <Route path='/fish/:name' component={FishDetails} />
```

Comment:
Be mindful of the 'name' property in the 'fish' table not being set as unique, meaning this could break if someone added a duplicate name.
Ordinarily we might make the route `/fish/:id`, but it's actually quite nice having the name visible in the address bar, so a better option might be to just update the database table so that uniqueness is guaranteed.

Reply:
resolved in #18